guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,465 @@
+
+Replace `com.mycompany.MainClass` with the actual main class of your application.
+
+Execute the following command to containerize and publish your application's Docker image:
+
+```bash
+mvn clean package docker:build
+```
+
+### Creating Kubernetes Service for Akka remoting
+
+Next we will declare the Akka remoting port we'd like to expose from our Pod using Kubernetes Service so it can be referenced from one Pod to another.
+
+Execute the following command to create Kubernetes Service for the application's Akka remoting port:
+
+```bash
+cat << EOF | kubectl create -f -
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "myapp"
+    },
+    "name": "myapp-akka-remoting"
+  },
+  "spec": {
+    "clusterIP": "None",
+    "ports": [
+      {
+        "port": 2551,
+        "protocol": "TCP",
+        "targetPort": 2551
+      }
+    ],
+    "selector": {
+      "app": "myapp"
+    }
+  }
+}
+EOF
+```
+
+This Kubernetes Service has `myapp-akka-remoting` as its name. The service exposes TCP port `2551` which is the application's Akka remoting port.
+
+All Pods that have the label `app` with the value of `myapp` will expose the TCP port `2551`.
+
+To view the list of Kubernetes services, use the following command:
+
+```bash
+kubectl get services
+```
+
+We can also filter the Kubernetes services based on our created label `app=myapp`:
+
+```bash
+kubectl get services --selector=app=myapp
+```
+
+### Creating Kubernetes StatefulSet resource
+
+Once we have the application containerized and published, and the Kubernetes Service declared for the application's Akka remoting port, we are ready to deploy the application.
+
+Execute the following command to deploy the application:
+
+```bash
+cat << EOF | kubectl create -f -
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "StatefulSet",
+  "metadata": {
+    "name": "myapp"
+  },
+  "spec": {
+    "serviceName": "myapp",
+    "replicas": 1,
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "myapp"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "myapp",
+            "image": "mygroup/myapp",
+            "imagePullPolicy": "Never",
+            "ports": [
+              {
+                "containerPort": 2551,
+                "name": "akka-remote"
+              }
+            ],
+            "resources": {
+              "limits": {
+                "cpu": "250m",
+                "memory": "384Mi"
+              },
+              "requests": {
+                "cpu": "250m",
+                "memory": "384Mi"
+              }
+            },
+            "env": [
+              {
+                "name": "AKKA_ACTOR_SYSTEM_NAME",
+                "value": "myapp-actor-system"
+              },
+              {
+                "name": "AKKA_REMOTING_BIND_PORT",
+                "value": "2551"
```

Comment:
I'm not sure how to `fieldRef` the port since the port itself is not present in the Pod's metadata.